src/dom.js

```diff
@@ -0,0 +1,152 @@
+// @flow
+
+import { Css } from './consts';
+
+export type Position = {| x: number, y: number |};
+
+function classNameToSet(el: HTMLElement): Set<string> {
```

Comment:
Is there a good way to add some unit test coverage for the functions in this file?

Reply:
Yes, it's simple enough: create `dom.test.js` and voila.  Should it be done in this PR or a new one though, @kujenga ?

Reply:
A new PR sounds good to me 👍

Reply:
voila = add test cases